mtools/mloginfo/sections/connection_section.py

```diff
@@ -51,6 +71,18 @@ def run(self):
                     ip, _ = tokens[3].split(':')
```

Comment:
[Optional] Not directly related to this change, but we need to consider a case where there is no element entry in the array. Attempting to access an index of array out of bound will produce `IndexError` exception, which should be handled